src/openrct2/peep/Staff.cpp

```diff
@@ -1791,8 +1791,8 @@ void Staff::Tick128UpdateStaff()
     SpriteType = newSpriteType;
     ActionSpriteImageOffset = 0;
     WalkingFrameNum = 0;
-    if (Action < PeepActionType::None1)
-        Action = PeepActionType::None2;
+    if (Action < PeepActionType::Idle)
+        Action = PeepActionType::Walking;
```

Comment:
How odd. Do you want me to check that in the assembly

Reply:
Would be great

Reply:
![image](https://user-images.githubusercontent.com/1277401/118161027-fe14fa80-b416-11eb-81d9-62ee35d39986.png)
looks like it is correct